### PR TITLE
Template code formatting

### PR DESCRIPTION
**Pull request by atnartur**
Added some template code formatting

### CODE REVIEW

**Comment by kamermans** [comment]
Your tests will pass now if you pull the latest changes on `master` (I would recommend rebasing it).

I would note, however, that as ugly as the indentation is in the template file, it was probably done intentionally to prevent generating nginx files with tons of random indentation.  If your PR is merged, the files that are produced will be very hard to follow, with sections indented seemingly at random.

**Comment by buchdag** [comment]
Closing for reasons mentioned by @kamermans
